common/persistence/sql/common.go

```diff
@@ -53,8 +53,8 @@ func (m *sqlStore) Close() {
 	}
 }
 
-func (m *sqlStore) txExecute(ctx context.Context, operation string, f func(tx sqlplugin.Tx) error) error {
-	tx, err := m.db.BeginTx(ctx)
+func (m *sqlStore) txExecute(dbShardID int, ctx context.Context, operation string, f func(tx sqlplugin.Tx) error) error {
```

Comment:
nit: As far as I know idiomatic Golang suggests passing context as the first argument (if present)

Reply:
Yeah not sure how I missed this one. Good catch. Thanks!